src/main/resources/assets/minecolonies/gui/windowfilterablelist.xml

```diff
@@ -1,26 +0,0 @@
-<window xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" size="190 244" pause="false"
```

Comment:
not sure about that

Reply:
Doesn't seem like it gets called anywhere

Reply:
"doesn't" doesn't seem sure

Reply:
I'm 95% sure it doesn't get called anywhere. Doing a search in files for `windowfilterablelist` has no results

Reply:
it's not used anywhere
![Screen Shot 2021-02-26 at 8 29 21 am](https://user-images.githubusercontent.com/28527281/109227891-c087d480-780c-11eb-9e26-0d1c909017be.png)

Reply:
Oh it's been moved to layouts